### PR TITLE
ref(TS) Remove ts-ignores

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Review by saghul — APPROVED** [review]

File: react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ import TogglePinToStageButton from './TogglePinToStageButton';
  * The type of the React {@code Component} props of
  * {@link LocalVideoMenuTriggerButton}.
  */
-interface Props extends WithTranslation {
+type Props = WithTranslation & {
```
Comment (saghul) [inline react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx:50]: why this change?

Comment (robertpin) [reply react/features/video-menu/components/web/LocalVideoMenuTriggerButton.tsx]: I was testing something and forgot to discard

**Review by saghul — APPROVED** [review]
